config.go

```diff
@@ -12,18 +13,39 @@ import (
 )
 
 // Config is used to store details used in generation of amortization table.
+// TODO: update readme for RoundingPlaces and RoundingErr Tolerance
+//
+//  Fields:
```

Comment:
add comments with fields themselves so that they show up on godoc with fields.
besides, easy to track: which field has been documented and which one hasn't.